### PR TITLE
Feature build on appveyor

### PR DESCRIPTION
**Pull request by omichel**
AppVeyor VM only has OpenGL 1.1, however it seems [possible](https://github.com/Wangscape/Wangscape/pull/121/files) to install a better OpenGL.
AppVeyor VM is Windows 2012 server and misses `C:\Windows\System32\XInput9_1_0.dll` (which is available since Windows Vista).

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by DavidMansolino** [comment]
:eyes:

File: src/packaging/publish_release.py [hunk 1]
```diff
@@ -67,13 +67,17 @@
 
 if not releaseExists:
     print('Creating release "%s" with tag "%s" on commit "%s"' % (title, tag, options.commit))
+    if tag:
+        draft = True
+    else:
+        draft = False
```
Comment (DavidMansolino) [inline src/packaging/publish_release.py]: You can replace these line by (but if you think it is more understandable like this it's fine for me too):
```
draft = True if tag else False
```

File: projects/robots/robotis/darwin-op/plugins/robot_windows/robotis-op2_window/SSH.cpp [hunk 2]
```diff
@@ -92,13 +92,13 @@ int SSH::openSSHSession(const QString &IPAddress, const QString &username, const
 int SSH::verifyKnownHost() {
   int state;
```
Comment (DavidMansolino) [inline projects/robots/robotis/darwin-op/plugins/robot_windows/robotis-op2_window/SSH.cpp]: The changes to this file are not compiling on Ubuntu 16.04.

Comment (DavidMansolino) [reply projects/robots/robotis/darwin-op/plugins/robot_windows/robotis-op2_window/SSH.cpp]: I did just launch a Travis test to make sure it is not only on my environment.

**Comment by omichel** [comment]
👍

**Pull request merged**